yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -171,28 +194,8 @@ func saveMigrationReportsFn(msr *metadb.MigrationStatusRecord) {
 	importDataReportFilePath := filepath.Join(backupDir, "reports", "import_data_report.txt")
 	strCmd = fmt.Sprintf("yb-voyager import data status -e %s > %q", exportDir, importDataReportFilePath)
 	importDataStatusCmd := exec.Command("bash", "-c", strCmd)
-	targetDBPassword, err = askPassword("target DB", "", "TARGET_DB_PASSWORD")
-	if err != nil {
-		utils.ErrExit("getting target db password: %v", err)
-	}
-	importDataStatusCmd.Env = append(os.Environ(), fmt.Sprintf("TARGET_DB_PASSWORD=%s", targetDBPassword))
-
-	if msr.FallForwardEnabled {
-		fallForwardDBPassword, err = askPassword("fall-forward DB", "", "FF_DB_PASSWORD")
-		if err != nil {
-			utils.ErrExit("getting fall-forward db password: %v", err)
-		}
-		importDataStatusCmd.Env = append(importDataStatusCmd.Env, fmt.Sprintf("FF_DB_PASSWORD=%s", fallForwardDBPassword))
-	}
-
-	if msr.FallbackEnabled {
-		sourceDBPassword, err = askPassword("fall-back DB", "", "SOURCE_DB_PASSWORD")
-		if err != nil {
-			utils.ErrExit("getting fall-back db password: %v", err)
-		}
-		importDataStatusCmd.Env = append(importDataStatusCmd.Env, fmt.Sprintf("SOURCE_DB_PASSWORD=%s", sourceDBPassword))
-	}
 
+	importDataStatusCmd.Env = append(os.Environ(), passwordsEnvVars...)
```

Comment:
I don't think we need passwords anymore for import data status. @priyanshi-yb pls confirm this.

Reply:
Yes, now the passwords are not required anymore in the import data status command.

Reply:
cool, thanks